tests/protocol/mockwiredata.py

```diff
@@ -239,6 +244,8 @@ def mock_http_get(self, url, *_, **kwargs):
                 content = self.vm_settings
                 response_headers = [('ETag', self.etag)]
             self.call_counts["vm_settings"] += 1
+        elif '{0}/metadata/compute'.format(IMDS_ENDPOINT) in url:
+            content = json.dumps(self.imds_info.get("compute", "{}"))
```

Comment:
Right now (perhaps always? not sure if this will ever change), this is the only type of IMDS call. Behavior if a new type of call is added to IMDS is to throw a `NotImplementedError`.

Reply:
we should probably return valid data, instead of empty

Reply:
never mind, i misread the code